packages/medusa/src/api/routes/admin/orders/create-fulfillment.ts

```diff
@@ -115,17 +115,27 @@ export default async (req, res) => {
       existingFulfillments.map((fulfillment) => [fulfillment.id, fulfillment])
     )
 
-    const { fulfillments } = await orderService
+    await orderService
       .withTransaction(transactionManager)
       .createFulfillment(id, validated.items, {
         metadata: validated.metadata,
         no_notification: validated.no_notification,
       })
 
-    const pvInventoryServiceTx =
-      pvInventoryService.withTransaction(transactionManager)
-
     if (validated.location_id) {
```

Comment:
suggestion(non blocking): We might take the time to migrate the route to use transformers middleware and cleanup a bit?